### PR TITLE
APEXMALHAR-2343 #resolve #comment Count Accumulation should only incr…

### PR DESCRIPTION
**Pull request by brightchen**
…ease one for each tuple

### CODE REVIEW

**Comment by brightchen** [comment]
@davidyan74 Please review

**Comment by davidyan74** [comment]
LGTM @siyuanh @chinmaykolhatkar please also review and make sure this does not break existing examples that make use of the Count accumulation.

File: library/src/main/java/org/apache/apex/malhar/lib/window/accumulation/Count.java [hunk 1]
```diff
@@ -26,7 +26,7 @@
  *
  * @since 3.5.0
  */
-public class Count implements Accumulation<Long, MutableLong, Long>
+public class Count<T> implements Accumulation<T, MutableLong, Long>
```
Comment (chinmaykolhatkar) [inline library/src/main/java/org/apache/apex/malhar/lib/window/accumulation/Count.java:29]: It doesn't make sense to parameterize Count class as the type of input really doesn't matter.

Comment (davidyan74) [reply library/src/main/java/org/apache/apex/malhar/lib/window/accumulation/Count.java:29]: That's true. Without this parameterization, the accumulation input type parameter needs to be a supertype of the input tuple type. I will issue a PR to reflect that.

Comment (davidyan74) [reply library/src/main/java/org/apache/apex/malhar/lib/window/accumulation/Count.java:29]: The PR is #517

**Comment by davidyan74** [comment]
This PR is superseded by #517. Please refer to that PR instead. This PR can be closed.

**Pull request closed without merging**